src/CommitmentsV2/SFA.DAS.CommitmentsV2/Application/Queries/GetDraftApprenticeships/GetDraftApprenticeshipsHandler.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using SFA.DAS.CommitmentsV2.Data;
+using SFA.DAS.CommitmentsV2.Types.Dtos;
+
+namespace SFA.DAS.CommitmentsV2.Application.Queries.GetDraftApprenticeships
+{
+    public class GetDraftApprenticeshipsHandler : IRequestHandler<GetDraftApprenticeshipsRequest, GetDraftApprenticeshipsResult>
+    {
+        private readonly Lazy<ProviderCommitmentsDbContext> _dbContext;
+
+        public GetDraftApprenticeshipsHandler(Lazy<ProviderCommitmentsDbContext> dbContext)
+        {
+            _dbContext = dbContext;
+        }
+
+        public Task<GetDraftApprenticeshipsResult> Handle(GetDraftApprenticeshipsRequest request, CancellationToken cancellationToken)
+        {
+            var cohort = _dbContext.Value.Cohorts
+                .Where(x => x.Id == request.CohortId)
+                .Select(x => new { Cohort = x, DraftApprenticeship = x.Apprenticeships})
```

Comment:
I think this query should still work without having to pull back the cohort data e.g.

```c#
.Select(x => new { DraftApprenticeships = x.Apprenticeships })
```